recipe/meta.yaml

```diff
@@ -12,23 +12,27 @@ source:
 
 build:
   merge_build_host: True  # [win]
-  number: 0
-  skip: True  # [win]
+  number: 1
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
   build:
-    - {{ compiler('c') }}        # [not win]
-    - {{ compiler('cxx') }}      # [not win]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Comment:
Is there a reason we don't want to use the default compiler on Windows?

Reply:
This is a reference to m2w64-toolchain, which includes the default compiler for R recipes.

Reply:
As far as a I understood this is only used as a lot of R recipes don't compile with the default compilers but isn't a requirement? My guess is that the build errors go away when using the default compilers.

Reply:
Now I understand better: https://ci.appveyor.com/project/conda-forge/r-arrow-feedstock/builds/25955436/job/klrd07nivr8ri90j :)